tests/acceptance/features/webUISharingInternalUsersToRoot/shareWithUsers.feature

```diff
@@ -239,55 +257,25 @@ Feature: Sharing files and folders with internal users
-      Then user "Alice Hansen" should be listed as "<expected-role>" in the collaborators list for folder "Sample,Folder,With,Comma" on the webUI
-      And user "Alice Hansen" should be listed as "<expected-role>" in the collaborators list for file "sample,1.txt" on the webUI
-      And user "Alice" should have received a share with these details:
-        | field       | value                    |
-        | uid_owner   | Brian                    |
-        | share_with  | Alice                    |
-        | file_target | /Sample,Folder,With,Comma |
-        | item_type   | folder                   |
-        | permissions | <permissions-folder>     |
-      And user "Alice" should have received a share with these details:
-        | field       | value              |
-        | uid_owner   | Brian              |
-        | share_with  | Alice              |
-        | file_target | /sample,1.txt      |
-        | item_type   | file               |
-        | permissions | <permissions-file> |
-      And as "Alice" these resources should be listed on the webUI
-        | entry_name               |
-        | Sample,Folder,With,Comma |
-        | sample,1.txt             |
-      Examples:
-        | set-role             | expected-role        | permissions-folder              | permissions-file       |
-        | Viewer               | Viewer               | read,share                      | read,share             |
-        | Editor               | Editor               | read,update,create,delete,share | read,update,share      |
-        | Advanced permissions | Advanced permissions | read                            | read                   |
 
   Scenario: file list view image preview in file share
     Given user "Alice" has uploaded file "testavatar.jpg" to "testavatar.jpg"
     And user "Alice" has shared file "testavatar.jpg" with user "Brian"
     When user "Brian" logs in using the webUI
     Then the preview image of file "testavatar.jpg" should be displayed in the file list view on the webUI
 
+
   Scenario: file list view image preview in file share when previews is disabled
     Given the property "disablePreviews" of "options" has been set to true in web config file
     And user "Alice" has uploaded file "testavatar.jpg" to "testavatar.jpg"
     And user "Alice" has shared file "testavatar.jpg" with user "Brian"
     When user "Brian" logs in using the webUI
     Then the preview image of file "testavatar.jpg" should not be displayed in the file list view on the webUI
 
+
   Scenario: sharing file after renaming it is possible
-    Given user "Alice" has logged in using the webUI
-    And user "Alice" has uploaded file with content "test" to "lorem.txt"
+    Given user "Alice" has uploaded file with content "test" to "lorem.txt"
+    And user "Alice" has logged in using the webUI
     And the user has renamed file "lorem.txt" to "new-lorem.txt"
```

Comment:
no need to rename, lets use the final name here

Reply:
I think renaming step is required here as the scenario is ```Scenario: sharing file after renaming it is possible`